src/widgets/datetime/DateTimeWidget.js

```diff
@@ -28,8 +31,11 @@ const DateTimeWidget = (props :WidgetProps) => {
   return (
     <DateTimePicker
         disabled={disabled || readonly}
+        format={format}
         id={id}
+        mask={mask}
         onChange={handleChange}
+        placeholder={placeholder}
         value={value} />
```

Comment:
missed passing ampm prop

Reply:
making a note here that we need to adjust the `DateTimePicker` in LUK to allow for passing `ampm` prop.